rdf-delta-server-extra/src/main/java/org/seaborne/delta/server/s3/PatchStorageS3.java

```diff
@@ -120,5 +124,25 @@ public RDFPatch fetch(Id key) {
     }
 
     @Override
-    public void delete(Id id) {}
+    public void delete(Id id) {
+        client.deleteObject(new DeleteObjectRequest(bucketName, idToKey(id)));
+    }
+
+    private void retry(int maxRetryCount, Runnable action) {
+        int retryCount = 0;
+
+        while (true) {
+            try {
+                action.run();
+            } catch (Exception ex) {
+                if (++retryCount <= maxRetryCount) {
+                    Log.warn(this, ("Action failed, but will retry " + (maxRetryCount - retryCount + 1) + " more times."), ex);
+                    Lib.sleep(1000);
```

Comment:
I didn't think that a sleep is what would really be wanted here...but I figured that it could be resolved via this PR. :)